survey/application/src/test/java/me/nalab/survey/application/service/find/SurveyFindServiceTest.java

```diff
@@ -46,9 +47,21 @@ void SURVEY_FIND_SERVICE_TEST() {
 
 		SurveyDto result = surveyFindService.findSurvey(surveyId);
 
+		boolean isAscendingOrder = result.getFormQuestionDtoableList().stream()
+			.map(FormQuestionDtoable::getOrder)
+			.reduce((prev, current) -> {
+				if(prev.compareTo(current) < 0) {
+					return current;
+				} else {
+					throw new RuntimeException();
+				}
+			})
+			.isPresent();
```

Comment:
```suggestion
		result.getFormQuestionDtoableList().stream().
		.map(FormQuestionDtoable::getOrder)
		.reduce((prve, current) -> assertTrue(prev < current));
```

이런방식은 안되나요? 
예외에서 RuntimeException을 던지면 어떤 값에서 에러가나오는지 판단하기가 힘들다?

Reply:
예외를 던지면 isAscendingOrder가 false가 되는상황이 없을거 같아요..!

Reply:
이렇게 해도 상관은 없을 것 같네요! 단점이라면 뒤에 몇개의 문제가 있는 지 한번의 테스트로 파악할 수 없다 정도?